XamlControlsGallery/ControlPages/PersonPicturePage.xaml

```diff
@@ -32,18 +26,15 @@
                 </StackPanel>
             </local:ControlExample.Options>
             <local:ControlExample.Xaml>
-                <Grid>
-                    <RichTextBlock x:Name="rtb" Style="{StaticResource XamlCodeRichTextBlockStyle}" Visibility="{x:Bind ProfileImageRadio.IsChecked, Mode=OneWay}">
-                        <Paragraph>&lt;PersonPicture ProfilePicture=&quot;https://docs.microsoft.com/en-us/windows/uwp/contacts-and-calendar/images/shoulder-tap-static-payload.png&quot; /&gt;</Paragraph>
-                    </RichTextBlock>
-                    <RichTextBlock x:Name="rtb1" Style="{StaticResource XamlCodeRichTextBlockStyle}" Visibility="{x:Bind DisplayNameRadio.IsChecked, Mode=OneWay}">
-                        <Paragraph>&lt;PersonPicture DisplayName=&quot;Jane Doe&quot; /&gt;</Paragraph>
-                    </RichTextBlock>
-                    <RichTextBlock x:Name="rtb2" Style="{StaticResource XamlCodeRichTextBlockStyle}" Visibility="{x:Bind InitialsRadio.IsChecked, Mode=OneWay}">
-                        <Paragraph>&lt;PersonPicture Initials=&quot;SB&quot; /&gt;</Paragraph>
-                    </RichTextBlock>
-                </Grid>
+                <x:String>
+                    &lt;PersonPicture $(ProfilePicture)$(DisplayName)$(Initials) /&gt;
```

Comment:
Should the sample be updating based on which radio button is selected? The code makes me think it should, but the app doesn't change its text.

Reply:
It should. The `$(ProfilePicture)`, `$(DisplayName)` and `$(Initials)` should be dynamically updating based on the radio button. Let me take a closer look.

Reply:
The markup says

    <local:ControlExampleSubstitution Key="ProfilePicture"
        IsEnabled="{x:Bind ProfileImageRadio.IsChecked.Value}"
         Value="..." />

and the IsEnabled should be changing as the selection changes.

Reply:
Not sure why that is, but I still see:
![image](https://user-images.githubusercontent.com/13142665/51641719-c15a2800-1f1b-11e9-90be-0d77da731823.png)

Reply:
Investigating.

Reply:
Duh. Missing `Mode=OneWay`  in the three `IsEnabled="{x:Bind …}"` attributes. Should be `IsEnabled="{x:Bind …, Mode=OneWay}"`.

Reply:
Fixed in latest commit.